apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.tsx

```diff
@@ -319,7 +321,15 @@ export const BrowsePools = ({ poolAdvertisements }: BrowsePoolsProps) => {
 };
 
 const BrowsePoolsApi = () => {
-  const [{ data, fetching, error }] = useBrowsePoolAdvertisementsQuery();
+  // get UTC time, in the appropriate format, from client
+  // https://stackoverflow.com/a/11964609
+  const now = new Date();
+  const nowUTC = new Date(now.getTime() + now.getTimezoneOffset() * 60000);
+  const formattedNowUTCRef = useRef(format(nowUTC, DATETIME_FORMAT_STRING)); // prevent infinite re-render
```

Comment:
Since this does not rely on any internal state, we could likely lift this outside of the component and not need the `useRef`. You could probably put something like the following in `date-helpers` then just import that 🤔 

```tsx
import { formatInTimeZone } from 'date-fns-tz'

const nowUTC = formatInTimeZone(new Date(), 'UTC', DATETIME_FORMAT_STRING);
```

Reply:
Hmm, yes, might as well. Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6479/commits/b172fbbb06c09bf3afb5088d444974b1ef876008